ykrt/src/compile/jitc_yk/trace_builder.rs

```diff
@@ -70,20 +71,38 @@ impl<'a> TraceBuilder<'a> {
             if inst.is_ptr_add() {
                 let op = inst.operand(0);
                 // unwrap safe: we know the AOT code was produced by ykllvm.
-                if trace_input
-                    .unwrap()
+                let ti_op = trace_input.unwrap();
+                if ti_op
```

Comment:
`if let Some(ti_op) = trace_input` ?

Reply:
The line above this change already justifies the unwrap. I didn't change that.

Reply:
OK.